var/spack/repos/builtin/packages/py-maestrowf/package.py

```diff
@@ -30,11 +30,16 @@ class PyMaestrowf(PythonPackage):
        simulation studies."""
 
     homepage = "https://github.com/LLNL/maestrowf/"
-    url      = "https://github.com/LLNL/maestrowf/archive/v1.0.1.tar.gz"
+    url      = "https://github.com/LLNL/maestrowf/archive/v1.1.2x.tar.gz"
```

Comment:
What's with the `x` at the end?

Reply:
You know, that's a good question. It appears like it might be a typo.

Reply:
@adamjstewart -- corrected.